src/ui/public/doc_table/components/table_row/details.html

```diff
@@ -1,6 +1,20 @@
 <td colspan="{{ columns.length + 2 }}">
-  <a class="pull-right" ng-href="#/doc/{{indexPattern.id}}/{{row._index}}/{{row._type}}/?id={{row._id | uriescape}}">
-    <small>Link to /{{row._index}}/{{row._type}}/{{row._id | uriescape}}</small></i>
-  </a>
+  <div class="documentTableRow__actions">
+    <a
+      class="btn btn-default btn-xs documentTableRow__action"
+      ng-href="{{ getContextAppHref() }}"
+      ng-if="indexPattern.hasTimeField()"
+    >
+      <i class="fa fa-files-o" />
```

Comment:
We should `<span>` instead of `<i>` for more semantic markup, and we can also use `kuiIcon` here instead of `fa`.